Makefile

```diff
@@ -79,7 +79,7 @@ IAMCTL_OUTPUT_FILE ?= iam_policy.go
 IAMCTL_GO_PACKAGE ?= awsloadbalancercontroller
 
 # File name of the generated CredentialsRequest CR.
-IAMCTL_OUTPUT_CR_FILE ?= ./hack/controller/controller-credentials-request.yaml
+IAMCTL_OUTPUT_CR_FILE ?= ./hack/controller/controller-credentials-request-minify.yaml
```

Comment:
Is this file name `minify` even though it isn't technically minified in release-1.0? Or it actually minified?

Reply:
It's actually minified. I changed the name of the file in `main` 1) to better reflect its contents, 2)  because 2 versions of the credentialsrequest are not published: minified and not minified.